app/src/main/java/com/github/ppartisan/simplealarms/ui/AlarmLandingPageFragment.java

```diff
@@ -1,45 +1,120 @@
 package com.github.ppartisan.simplealarms.ui;
 
+import android.content.Context;
 import android.content.Intent;
+import android.hardware.Sensor;
+import android.hardware.SensorEvent;
+import android.hardware.SensorEventListener;
+import android.hardware.SensorManager;
 import android.os.Bundle;
+import android.os.Handler;
+import android.os.PowerManager;
+import android.os.Vibrator;
 import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.Button;
+import android.widget.TextView;
 
 import com.github.ppartisan.simplealarms.R;
 
-public final class AlarmLandingPageFragment extends Fragment implements View.OnClickListener {
 
+import static android.content.Context.VIBRATOR_SERVICE;
+
+public final class AlarmLandingPageFragment extends Fragment implements SensorEventListener {
+    public  long lastTime;
```

Comment:
These scope of these can be reduced - `private` is fine.

Reply:
@PPartisan  I fixed it